package.json

```diff
@@ -27,7 +27,7 @@
     "@aws-sdk/client-iam": "^3.53.0",
     "@aws-sdk/client-s3": "^3.53.1",
     "argparse": "^1.0.10",
-    "auspice": "2.40.0",
+    "auspice": "github:nextstrain/auspice#aa150e3",
```

Comment:
@victorlin do you know why this is failing? It uses the same approach as the automatic PRs made from the auspice repo (https://github.com/nextstrain/auspice/blob/master/.github/workflows/make_pr_for_downstream_repo.yaml#L59)

```
Run npm ci
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/nextstrain/auspice.git
npm ERR! 
npm ERR! Warning: Permanently added the ECDSA host key for IP address '140.82.114.4' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128
```

Reply:
It also seems to have worked in the past (e.g. https://github.com/nextstrain/nextstrain.org/pull/533), so I wonder if this is a GitHub action change or we need to add SSH credentials?

Reply:
The heroku review app also failed, with some more helpful logs:

```
       npm ERR! Error while executing:
       npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/nextstrain/auspice.git
       npm ERR! 
       npm ERR! Host key verification failed.
       npm ERR! fatal: Could not read from remote repository.
       npm ERR! 
       npm ERR! Please make sure you have the correct access rights
       npm ERR! and the repository exists.
       npm ERR! 
       npm ERR! exited with error code: 128
```

Reply:
@joverlee521 that sounds right. @jameshadfield it looks like you used a newer version of NPM during `npm install` which upgraded the lockfile version. I'll regenerate with npm 6.14.15 which should fix it.

Reply:
Aha - another gotcha. Thanks both!

I knew I was using a newer version of node/npm than we want for this repo as we don't want to get the nasy lockfile-version-changes, but didn't realise it would have flow on effects such as this!

Reply:
Not your fault, it's too easy for this to happen! Created a separate issue: #620